avocado/utils/asset.py

```diff
@@ -299,6 +299,8 @@ def _has_valid_hash(cls, asset_path, asset_hash=None):
         If asset_hash is None then will consider a valid asset.
         """
         if asset_hash is None:
+            LOG.info("No hash provided. Cannot ensure the asset"
```

Comment:
Hi @wainersm, I would use `warning` instead of `info`, what do you think?

Reply:
+1 for warning

Reply:
I agree @richtja , thanks for the suggestion. I am going to send a updated version soon.